contrib/direnv

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# Look for the .config folder in the current directory and up
+find_prj_config() (
```

Comment:
Minor thing, just to make sure I'm on the right page, is this just for if we can't detect this via Git?

If so, perhaps worth making this more explicit with;

```suggestion
# Discover PRJ_ROOT if not in a git repository.
# Look for the .config folder in the current directory.
find_prj_root_without_git() (
```

you might even want a simple `find_prj_root_with_git` for parity.

Reply:
Addressed in 3ea5e756ca4c228aeb8586eb13fa899e295fdca6